test/plugins/link.js

```diff
@@ -4,10 +4,14 @@ var assert = require("assert");
 
 var util = require("../util");
 var link = require("../../src/plugins/irc-events/link.js");
+const path = require("path");
 
 describe("Link plugin", function() {
 	before(function(done) {
 		this.app = util.createWebserver();
+		this.app.get("/favicon.png", function(req, res) {
+			res.sendFile(path.resolve(__dirname, "../../client/img/apple-touch-icon-120x120.png"));
```

Comment:
Our prefetcher fetches favicons as a thumbnail?! At https://github.com/thelounge/lounge/pull/1276/files#diff-4805d608cbc31851a7bee1bf4c7e247eR54 I'm only seeing `og:image` and `twitter:image`, what am I missing?